<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZkpForm</class>
 <widget class="QWidget" name="ZkpForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#ZkpForm{
background-color: white;
}

QGroupBox{
background-color: transparent;
}
#scrollAWC,#scrollArea
{
   background:transparent;
}

QGroupBox::title {
   subcontrol-origin: margin;
   subcontrol-position: top center;
   padding: 0 3px;
   font-weight: bold;
   color : blue;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>3</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAWC">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>861</width>
        <height>430</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>15</number>
       </property>
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="grpb_burn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Burn Coin And Gen ZKP</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <property name="spacing">
               <number>12</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_10">
                <property name="styleSheet">
                 <string notr="true">color:red; font-weight :bold;</string>
                </property>
                <property name="text">
                 <string>*</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_9">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>Public Key:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_b_pubkey">
                <property name="toolTip">
                 <string>Public key for redemption burned coin</string>
                </property>
                <property name="maxLength">
                 <number>34</number>
                </property>
                <property name="placeholderText">
                 <string>Public key for redemption burned coin</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pb_newPubKey">
                <property name="text">
                 <string>New</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="spacing">
               <number>12</number>
              </property>
              <item>
               <widget class="QLabel" name="label_11">
                <property name="styleSheet">
                 <string notr="true">color:red; font-weight :bold;</string>
                </property>
                <property name="text">
                 <string>* 1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lab_burn">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>Burn Amount:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spb_BurnAmount">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="suffix">
                 <string> VASH</string>
                </property>
                <property name="minimum">
                 <number>1000</number>
                </property>
                <property name="maximum">
                 <number>1000</number>
                </property>
                <property name="singleStep">
                 <number>1000</number>
                </property>
                <property name="value">
                 <number>1000</number>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_13">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_19">
                  <property name="styleSheet">
                   <string notr="true">color:red; font-weight :bold;</string>
                  </property>
                  <property name="text">
                   <string>* 3</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pb_burn">
                  <property name="text">
                   <string>Burn</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="lab_burnProof">
                <property name="text">
                 <string>ZKP Hash:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_burnProof">
                <property name="toolTip">
                 <string>Please remember this hash code</string>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
                <property name="placeholderText">
                 <string>Please remember this hash code</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="grpb_gmk">
           <property name="title">
            <string>Gen Mint Key</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="spacing">
               <number>12</number>
              </property>
              <item>
               <widget class="QLabel" name="label_12">
                <property name="styleSheet">
                 <string notr="true">color:red; font-weight :bold;</string>
                </property>
                <property name="text">
                 <string>*</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>ZKP Hash:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_zkpForMintKey">
                <property name="placeholderText">
                 <string>Input your ZKP hash</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <property name="spacing">
               <number>12</number>
              </property>
              <item>
               <widget class="QLabel" name="label_13">
                <property name="styleSheet">
                 <string notr="true">color:red; font-weight :bold;</string>
                </property>
                <property name="text">
                 <string>* 2</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>To Address:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_g_toAddress">
                <property name="toolTip">
                 <string>Input Receive Coin Address, we recommend you use new address</string>
                </property>
                <property name="maxLength">
                 <number>34</number>
                </property>
                <property name="placeholderText">
                 <string>Input Receive Coin Address, we recommend you use new address</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pb_g_new">
                <property name="text">
                 <string>New</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <property name="spacing">
               <number>12</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_14">
                <property name="styleSheet">
                 <string notr="true">color:red; font-weight :bold;</string>
                </property>
                <property name="text">
                 <string>*</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>Mint Amount:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="sb_gMintAmount">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="suffix">
                 <string> VASH</string>
                </property>
                <property name="minimum">
                 <number>1000</number>
                </property>
                <property name="maximum">
                 <number>1000</number>
                </property>
                <property name="singleStep">
                 <number>1000</number>
                </property>
                <property name="value">
                 <number>1000</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pb_genKey">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Gen</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Mint Key:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_gMintKey">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QGroupBox" name="grpb_mint">
           <property name="title">
            <string>Mint</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>9</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <property name="spacing">
               <number>12</number>
              </property>
              <item>
               <widget class="QLabel" name="label_15">
                <property name="styleSheet">
                 <string notr="true">color:red; font-weight :bold;</string>
                </property>
                <property name="text">
                 <string>*</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_5">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>Mint Key:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_m_zkpHash">
                <property name="placeholderText">
                 <string>Input your mint key</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_14">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_20">
                  <property name="styleSheet">
                   <string notr="true">color:red; font-weight :bold;</string>
                  </property>
                  <property name="text">
                   <string>* 4</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pb_Mint">
                  <property name="text">
                   <string>Mint</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_16">
                <property name="styleSheet">
                 <string notr="true">color:red; font-weight :bold;</string>
                </property>
                <property name="text">
                 <string>*</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_6">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>To Address:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_m_toAddress">
                <property name="maxLength">
                 <number>34</number>
                </property>
                <property name="placeholderText">
                 <string>Input Receive Coin Address, we recommend you use new address</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_17">
                <property name="styleSheet">
                 <string notr="true">color:red; font-weight :bold;</string>
                </property>
                <property name="text">
                 <string>*</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_7">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>Mint Key:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_m_MintKey">
                <property name="placeholderText">
                 <string>Input your mint key</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <property name="spacing">
               <number>12</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_18">
                <property name="styleSheet">
                 <string notr="true">color:red; font-weight :bold;</string>
                </property>
                <property name="text">
                 <string>*</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_8">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>Mint Amount:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="sb_m_amount">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="suffix">
                 <string> VASH</string>
                </property>
                <property name="minimum">
                 <number>1000</number>
                </property>
                <property name="maximum">
                 <number>1000</number>
                </property>
                <property name="singleStep">
                 <number>100</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <property name="topMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="lab_err">
                <property name="styleSheet">
                 <string notr="true">color:red;    font-weight: bold;</string>
                </property>
                <property name="text">
                 <string>Error msg.</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
